kibana.md

```diff
@@ -107,18 +107,69 @@ Notice how we need to type in the format `file://` + absolute path (yes, with th
 
 If you are using Kibana 7.9.x or newer, you need **an extra post-installation step**. This will slightly modify some core Kibana files.
 
+#### Before Kibana 8.15.0
+
+```bash
+# Patch Kibana core files 
+node/bin/node plugins/readonlyrestkbn/ror-tools.js patch
+```
+
+#### For Kibana 8.15.0 and never
+
+##### For Linux 
+
 ```bash
 # Patch Kibana core files 
-$ node/bin/node plugins/readonlyrestkbn/ror-tools.js patch
+node/glibc-217/bin/node plugins/readonlyrestkbn/ror-tools.js patch
+```
+
+##### For macOS
+
+```bash
+# Patch Kibana core files 
```

Comment:
Let's remove the comment. It doesn't help, but there is a problem with coping the content using the copy button rendered by our docs